README.md

```diff
@@ -10,8 +10,10 @@ Includes functions for posterior analysis, model checking, comparison and diagno
 
 ## Documentation
 
-The official ArviZ documentation can be found here
-https://arviz-devs.github.io/arviz/index.html
+The ArviZ documentation can be found in the [official docs](https://arviz-devs.github.io/arviz/index.html) First time users may find the [quickstart](https://arviz-devs.github.io/arviz/notebooks/Introduction.html).
+to be helpful. Additional guidance can be found in the
+[usage documentation](https://arviz-devs.github.io/arviz/usage.html)
```

Comment:
```suggestion
[usage documentation](https://arviz-devs.github.io/arviz/usage.html).
```

Reply:
Ugh getting sloppy. Thank you. After this I'm done for tonight :)

Reply:
:+1: i can merge in a few minutes -- thanks much!